examples/tuners/ga_customer_tuner/customer_tuner.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 
 from nni.tuner import Tuner
+from nni.utils import *
```

Comment:
suggest to import only required function, avoid import *

Reply:
Done